src/ng/directive/ngShowHide.js

```diff
@@ -89,8 +89,10 @@ var NG_HIDE_IN_PROGRESS_CLASS = 'ng-hide-animate';
  * property to block during animation states--ngAnimate will handle the style toggling automatically for you.
  *
  * @animations
- * addClass: `.ng-hide` - happens after the `ngShow` expression evaluates to a truthy value and the just before contents are set to visible
- * removeClass: `.ng-hide` - happens after the `ngShow` expression evaluates to a non truthy value and just before the contents are set to hidden
+ * | Animation                        | Occurs                              |
+ * |----------------------------------|-------------------------------------|
+ * | {@link $animate#addClass addClass} `.ng-hide`  | after the `ngShow` expression evaluates to a truthy value and the just before contents are set to visible |
```

Comment:
It's the other way around: `removeClass` when truthy, `addClass` when falsy.
(There is also a similat section in `ngHide` (also wrong), that could use a table :smiley:)